packages/builders/src/jest/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Jest Target",
+  "description": "Uses Jest to run unit tests",
+  "type": "object",
+  "properties": {
+    "jestConfig": {
+      "type": "string",
+      "description": "Path to the Jest Configuration"
+    },
+    "tsConfig": {
+      "type": "string",
+      "description": "TsConfig Path"
+    },
+    "main": {
+      "type": "string",
+      "description": "Main Path"
```

Comment:
Could you provide a more information description? What is the main path used for?